readme.md

```diff
@@ -154,7 +154,7 @@ Additional global variables your code accesses during execution.
 
 Type: `array`
 
-Some [paths](https://github.com/sindresorhus/xo/blob/4a0db396766118d7918577d759cacb05cd99a354/index.js#L14-L20) are ignored by default. Additional ignores can be added here.
+Some [paths](https://github.com/sindresorhus/xo/blob/master/index.js#L10-L19) are ignored by default. Additional ignores can be added here.
```

Comment:
> master

Better to use an actual commit hash as this way it's going to break when we add more code above that array definition.